### PR TITLE
Document installation on VirtualBox

### PR DESCRIPTION
**Pull request by guyonvarch**
See the trello card: https://trello.com/c/uKTMzzdN/648-document-playos-installation-on-virtualbox

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -56,6 +53,36 @@ make develop
...
+
+1. Build the installer with:
+
+```sh
+nix-build \
+  --arg buildInstaller true \
+  --arg buildBundle false \
+  --arg buildLive false \
+  --arg buildDisk false
+```
```
Comment (knuton) [inline Readme.md]: Maybe a reference to the build arguments suffices for [self-help](https://en.wiktionary.org/wiki/give_a_man_a_fish_and_you_feed_him_for_a_day;_teach_a_man_to_fish_and_you_feed_him_for_a_lifetime)? https://github.com/dividat/playos/tree/develop#quick-start

Optionally adding a sub-heading to the arguments section to make it linkable might help.

File: Readme.md [hunk 2]
```diff
@@ -56,6 +53,36 @@ make develop
...
+
+- `Settings > Display > Graphics controller:` set `VBoxSVGA` (see [this issue](https://discourse.nixos.org/t/trying-to-fix-very-poor-virtualbox-install-experience/2488), but it [should be resolved on NixOS 20.03](https://github.com/NixOS/nixpkgs/commit/58d0134da072548eb66d9313ad629e4dffddfd9d)),
+- `Settings > System > Motherboard`: enable EFI.
+
+3. Start the virtual machine. You will be prompted to select an optical drive,
+   select the previously built `result/playos-installer-VERSION.iso`. Then,
+   follow the installation procedure. Once the installation has been completed,
+   power off the virtual machine.
+
+4. In `Settings > Storage`, remove the optical drive containing
```
Comment (knuton) [inline Readme.md]: Points 3 and 4 are general points for installing most any Linux from an ISO.

**Comment by guyonvarch** [comment]
> This is a good how-to, I just wonder whether it is too specific for the top-level Readme.
> 
> I left some comments with suggestions for focusing on the tricky bits. It also seems fine to leave the text as is, but I would consider moving it to a separate file and linking to it from the Readme then.

I agree, I moved it to `arch/Readme.md`, which has a section on testing on virtual machines. See [the live doc](https://github.com/guyonvarch/playos/tree/document-virtualbox-install/docs/arch#installation-on-virtualbox).

File: Readme.md (hunk 2 above)
Comment (guyonvarch) [reply Readme.md]: I had some trouble being stuck rebooting on the ISO. I proposed a simplified version on those steps and with this indication.

Note: I am not used to install systems with VirtualBox.

**Comment by knuton** [comment]
> I agree, I moved it to `arch/Readme.md`, which has a section on testing on virtual machines. See [the live doc](https://github.com/guyonvarch/playos/tree/document-virtualbox-install/docs/arch#installation-on-virtualbox).

Yeah, good call.

Small thing: The list of steps is disrupted, causing it to render as 1-2-1 instead of 1-2-3. Maybe you can fix that.

**Comment by guyonvarch** [comment]
> Small thing: The list of steps is disrupted, causing it to render as 1-2-1 instead of 1-2-3. Maybe you can fix that. 

That’s fixed!

**Pull request merged**